From 6e6c57bacc0072c0ca152280c7a37340be76787a Mon Sep 17 00:00:00 2001
Message-Id: <6e6c57bacc0072c0ca152280c7a37340be76787a@dist-git>
From: Michal Privoznik <mprivozn@redhat.com>
Date: Thu, 15 Dec 2016 18:17:46 +0100
Subject: [PATCH] qemuDomainAttachNetDevice: Enable multiqueue for vhost-user

RHEL-7.4: https://bugzilla.redhat.com/show_bug.cgi?id=1386976
RHEL-7.3.z: https://bugzilla.redhat.com/show_bug.cgi?id=1404186

We have everything ready. Actually the only limitation was our
check that denied hotplug of vhost-user.

Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
(cherry picked from commit 21db4ab0528ca6c78744148ca4b7515aaeb4d0bf)
Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
---
 src/qemu/qemu_hotplug.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/qemu/qemu_hotplug.c b/src/qemu/qemu_hotplug.c
index 573cb8c93..7e4a653e6 100644
--- a/src/qemu/qemu_hotplug.c
+++ b/src/qemu/qemu_hotplug.c
@@ -926,7 +926,8 @@ qemuDomainAttachNetDevice(virQEMUDriverPtr driver,
         !(actualType == VIR_DOMAIN_NET_TYPE_NETWORK ||
           actualType == VIR_DOMAIN_NET_TYPE_BRIDGE ||
           actualType == VIR_DOMAIN_NET_TYPE_DIRECT ||
-          actualType == VIR_DOMAIN_NET_TYPE_ETHERNET)) {
+          actualType == VIR_DOMAIN_NET_TYPE_ETHERNET ||
+          actualType == VIR_DOMAIN_NET_TYPE_VHOSTUSER)) {
         virReportError(VIR_ERR_CONFIG_UNSUPPORTED,
                        _("Multiqueue network is not supported for: %s"),
                        virDomainNetTypeToString(actualType));
-- 
2.11.0

